service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/NonConsultationResourceMapper.java

```diff
@@ -1,6 +1,7 @@
 package uk.nhs.adaptors.gp2gp.ehr.mapper;
 
 import com.github.mustachejava.Mustache;
+import io.micrometer.common.util.StringUtils;
```

Comment:
Strange import.

Reply:
Fixed, must have selected the incorrect import in error.